scripts/train/yamls/hf_causal_lm/mpt-7b_dolly_sft.yaml

```diff
@@ -0,0 +1,126 @@
+max_seq_len: 2048
+global_seed: 17
+
+# Run Name
+run_name: # If left blank, will be read from env var $COMPOSER_RUN_NAME
+
+model:
+  name: hf_causal_lm
+  device: cpu
+  pretrained: true
+  pretrained_model_name_or_path: mosaicml/mpt-7b
+  config_overrides:
+    attn_impl: triton
```

Comment:
this is no longer the correct way to do this, and actually is not possible until https://github.com/mosaicml/llm-foundry/pull/90 goes in

Reply:
Noted in the PR description. I think I am fine just waiting for #90 so that we can default to finetuning with Triton